programs/server/Server.cpp

```diff
@@ -697,7 +697,7 @@ try
     {
         const String config_path = config().getString("config-file", "config.xml");
         const auto config_dir = std::filesystem::path{config_path}.replace_filename("openssl.conf");
-        setenv("OPENSSL_CONF", config_dir.string(), true);
+        setenv("OPENSSL_CONF", config_dir.string().c_str(), true);
```

Comment:
I believe there is `c_str()` method in `path`
```suggestion
        setenv("OPENSSL_CONF", config_dir.c_str(), true);
```